depthai/src/main/java/org/bytedeco/depthai/presets/depthai.java

```diff
@@ -324,6 +324,8 @@ public void map(InfoMap infoMap) {
                .put(new Info("std::function<void(std::string,std::shared_ptr<ADatatype>)>").valueTypes("NameMessageCallback"))
                .put(new Info("std::function<void(std::shared_ptr<ADatatype>)>").valueTypes("MessageCallback"))
                .put(new Info("std::function<void()>").valueTypes("Callback"))
+
+               .put(new Info("dai::DeviceInfo::toString").javaText("public native @StdString String toString();"))
```

Comment:
BTW, `dai::DeviceInfo::toString()` doesn't exist in main branch. Is something new in the branch for Android?

Reply:
Yeah, some improvements on that branch, but they got left behind for now, until mainlined

Reply:
Added because it was failing otherwise with an overload between DeviceInfo toString and base class Pointer...